ref/general/oidc.adoc

```diff
@@ -0,0 +1,70 @@
+
+The client application here operates as an OpenID Connect Relying Party (RP); and relies on Open Liberty as the OpenID Connect Provider (OP) to do its user authentication. The RP receives user identity information from the OP and passes the identity information in the ID token to the back-end Liberty application server in the HTTP Request header. The application server can extract user identity information from the HTTP header to perform further authorization.
+
+Some of the benefits of using Liberty as the OP are, Liberty OP can be integrated with the Liberty LDAP user registry.(linking), Liberty OP can be configured as token services by delegating user authentication to SAML identity provider or another OpenID Connect provider, Liberty OP is highly customizable, and you can customize the claims in ID Token, userinfo, login form, consent form.
+
+
+== Configuring Open Liberty as an SSO provider
+
+You can configure a Liberty server to act as an OpenID Connect Provider by enabling the openidConnectServer-1.0 feature in Liberty.
+
+The OpenID Connect Provider (openidConnectServer-1.0) feature can be configured to manage user authentication for use by OpenID Connect clients and microservices. The OpenID Connect Provider can work directly with a user registry such as LDAP or act as an identity broker and delegate to another security provider.
+
+Add the `openidConnectServer-1.0` Liberty feature and any other needed features to the `server.xml` file. The `transportSecurity-1.0` feature is also required for the `openidConnectServer-1.0` feature.
+
+[source, java]
+----
+<feature>openidConnectServer-1.0</feature>
+<feature>transportSecurity-1.0</feature>
+----
+
+Define an `OAuth` service provider. The configuration of an `OAuth` service provider includes the appropriate `oauth-roles`, `oauthProvider`, and user registry elements. Any user that is authorized to use OpenID Connect must also be mapped to the authenticated `oauth-role`.
+
+In the following example, the OP expects the client's SSL port to be set to 443.
+
+[source, java]
+----
+<openidConnectProvider id="OidcConfigSample" oauthProviderRef="OAuthConfigSample" />
+
+<oauthProvider id="OAuthConfigSample">
+<localStore>
+<client name="client01" secret="{xor}LDo8LTor"
+displayname="client01"
+scope="openid profile email">
+   <redirect>https://server.example.com:443/oidcclient/redirect/client01</redirect>
+   <redirect>https://server2.example.com:443/oidcclient/redirect/client01</redirect>
+</client>
+</localStore>
+</oauthProvider>
+----
+
+Add an `openidConnectProvider` element whose `oauthProviderRef` attribute references the configured `oauthProvider`. Each `oauthProvider` can be referenced only by one `openidConnectProvider`, and two or more `openidConnectProvider` elements cannot reference to the same `oauthProvider`. The name attribute and the secret attribute of the client element must match the client ID and the client secret of the corresponding OpenID Connect Client.
```

Comment:
don't use "whose" for element, instead 
Add an `openidConnectProvider` element with the `oauthProviderRef` attribute specified to reference...

change "only by one"  to "by only one.." Split this sentence into two at the "and".
"by only one `openidConnectProvider`. Two or more `openidConnectProvider` elements cannot..."